src/promptflow-core/promptflow/core/_flow.py

```diff
@@ -82,6 +83,8 @@ def load(
         flow_dir, flow_filename = resolve_flow_path(source)
         flow_path = flow_dir / flow_filename
         if is_prompty_flow(flow_path):
+            if getattr(cls, "__call__", None) and inspect.iscoroutinefunction(AsyncFlow.__call__):
```

Comment:
why inspect.iscoroutinefunction(**AsyncFlow**.__call__)?

Reply:
change to cls.__call__